website/django/data/views.py

```diff
@@ -105,6 +108,10 @@ def index(request):
     if search_term:
         query, synonyms_count = apply_search(query, search_term, quotes=quotes, release=release)
 
+    if not show_deleted and not release:
```

Comment:
do we need to support cases where users want to see deleted variants for a specific release?